backend/src/packages/chaiNNer_standard/image_utility/compositing/z_stack.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List, Union
+
+import numpy as np
+
+from nodes.group import group
+from nodes.properties.inputs import EnumInput, ImageInput
+from nodes.properties.outputs import ImageOutput
+from nodes.utils.utils import ALPHABET, get_h_w_c
+
+from .. import compositing_group
+
+
+class Expression(Enum):
+    MEDIAN = "median"
+    MEAN = "mean"
+    MIN = "minimum"
+    MAX = "maximum"
+
+
+@compositing_group.register(
+    schema_id="chainner:image:z_stack",
+    name="Z-Stack Images",
+    description="""Aligns multiple images and evaluates them in relation to each other.""",
+    icon="BsLayersHalf",
+    inputs=[
+        EnumInput(Expression),
+        ImageInput("Image A"),
+        ImageInput("Image B"),
+        group("optional-list")(
+            *[
+                ImageInput(f"Image {letter}").make_optional()
+                for letter in ALPHABET[2:14]
+            ],
+        ),
+    ],
+    outputs=[ImageOutput()],
+)
+def z_stack(
+    expression: Expression,
+    *args: Union[np.ndarray, None],
+) -> np.ndarray:
+    """Align and evaluate images"""
+
+    inputs: List[Union[np.ndarray, None]] = [*args]
+    images = [x for x in inputs if x is not None]
```

Comment:
Just cause it's bothering me. 
```suggestion
    *inputs: np.ndarray | None,
) -> np.ndarray:
    """Align and evaluate images"""

    images = [x for x in inputs if x is not None]
```